velox/dwio/common/ParallelUnitLoader.cpp

```diff
@@ -55,7 +55,9 @@ class ParallelUnitLoader : public UnitLoader {
       std::vector<std::unique_ptr<LoadUnit>> units,
       folly::Executor* ioExecutor,
       uint16_t maxConcurrentLoads)
-      : loadUnits_(std::move(units)),
+      : loadUnits_(
+            std::make_move_iterator(units.begin()),
```

Comment:
can we use std::move() here?

Reply:
we can't because we need to convert each element from unique_ptr<LoadUnit> to shared_ptr<LoadUnit>